packages/contracts/docs/core/02-how-to-guides/02-dao-upgrading/index.md

```diff
@@ -6,4 +6,4 @@ title: DAO Upgrading
 
 To migrate an existing Aragon Client DAO read our guide about [Migrating your Aragon Client DAO](01-aragon-client-migration.md).
 
-AragonOS 6 makes future protocol upgrades much easier for you. To read about it visit our [Upgrading to Future aragonOS Releases](02-protocol-upgrades.md) section.
+AragonOS makes future protocol upgrades much easier for you. To read about it visit our [Upgrading to Future aragonOS Releases](02-protocol-upgrades.md) section.
```

Comment:
In one of the charts we already have the X, leaving for later?

Reply:
Fixed in 55a3dacf72b956668efec6d539506c70fc20f417

Reply:
There is another one that I found before then

Reply:
Can you tell me which file exactly?

Reply:
I globally searched and cannot find it